chia/full_node/weight_proof.py

```diff
@@ -618,6 +618,9 @@ def get_fork_point(self, received_summaries: List[SubEpochSummary]) -> uint32:
         for idx, summary_height in enumerate(ses_heights):
             log.debug(f"check summary {idx} height {summary_height}")
             local_ses = self.blockchain.get_ses(summary_height)
+            if idx == len(received_summaries) - 1:
```

Comment:
Could add `assert len(received_summaries) > 0` at the start of `get_fork_point`

Reply:
thats always the case, or we would have failed validation before this method
do you think it will be more readable that way?

Reply:
No, I'm happy with your judgement.

I just always want to point out any issue I can think of, and I don't know enough about this part yet to know if `len == 0` is possible.